stripe-patrons-data/src/main/scala/com/gu/patrons/lambdas/PatronCancelledEventLambda.scala

```diff
@@ -125,10 +142,21 @@ object PatronCancelledEventLambda extends StrictLogging {
     EitherT.fromEither(decode[PatronCancelledEvent](payload).left.map(e => InvalidJsonError(e.getMessage)))
   }
 
-  def getCustomer(config: PatronsStripeConfig, customerId: String): EitherT[Future, Error, Customer] = {
+  def getCustomer(config: PatronsStripeConfig, customerId: String): EitherT[Future, Error, StripeCustomer] = {
     logger.info(s"Attempting to retrieve customer with id $customerId from Stripe")
-    Stripe.apiKey = config.apiKey
-    EitherT.fromEither(Try(Customer.retrieve(customerId)).toEither.left.map(e => UserNotFoundStripeError(e.getMessage)))
+    val stripeService = new PatronsStripeService(config, runner)
+    val futureCustomer = stripeService.getCustomer(customerId).transform {
```

Comment:
I don't understand if the Failure is being handled why this needs to be wrapped in a Try?

Reply:
Because transform maps from a Try to a Try, it just allows us to use a custom error type for easier handling later on.